weed/command/filer_sync.go

```diff
@@ -65,6 +67,8 @@ func init() {
 	syncOptions.bProxyByFiler = cmdFilerSynchronize.Flag.Bool("b.filerProxy", false, "read and write file chunks by filer B instead of volume servers")
 	syncOptions.aDebug = cmdFilerSynchronize.Flag.Bool("a.debug", false, "debug mode to print out filer A received files")
 	syncOptions.bDebug = cmdFilerSynchronize.Flag.Bool("b.debug", false, "debug mode to print out filer B received files")
+	syncOptions.aOffset = cmdFilerSynchronize.Flag.Int("a.offset", 0, "synchronization from offset on filer A")
```

Comment:
Not clear what's the unit.

How about using a relative time similar to the "timeAgo" in other places?

Reply:
Thank you for reminding me.

I do need to explain the unit. The unit is seconds in this code and i want to change it to a millisecond timestamp.  

There is also something wrong with my naming. Many people don't know that offset is a timestamp.

I want to change it like this.
Eg.  **a.syncFromTs,  b.syncFromTs** 

It is similar to the consumption point of rocketmq: **CONSUME_FROM_TIMESTAMP**



I still don't recommend using "TimeAge".
We usually know the start time of synchrionization before filer.sync starts. 
This will result in an inaccurate synchronization point.